exdreader.py

```diff
@@ -167,14 +168,12 @@ def read_file(self, offset: int):
         file_info_bytes = self.file.read(24)
         file_info = SqPackFileInfo(file_info_bytes, offset)
         data: list[bytes] = []
-        match file_info.type:
-            case SqPackFileType.Empty:
-                raise Exception('File located at 0x' + hex(offset) + ' is empty.')
-            case SqPackFileType.Standard:
-                data = self.read_standard_file(file_info)
-            case _:
-                raise Exception('Type: ' + str(file_info.type) + ' not implemented.')
-        return data
+        if file_info.type == SqPackFileType.Empty:
+            raise Exception(f'File located at 0x{hex(offset)} is empty.')
+        elif file_info.type == SqPackFileType.Standard:
+            return self.read_standard_file(file_info)
```

Comment:
Will still like to keep it to assign to data and return data instead, as there is multiple types that will be added and at a later point extended to have a binary reader overhead for turning things into each sub category dependent on what is needed.

Reply:
That's fair. I'll revert back.